paprika_recipes/commands/download_recipes.py

```diff
@@ -25,6 +33,23 @@ def handle(self) -> None:
         for recipe in track(
             remote, total=remote.count(), description="Downloading Recipes"
         ):
+            if self.options.download_images:
+                images_path = self.options.export_path / "images"
+                images_path.mkdir(parents=True, exist_ok=True)
+
+                image = remote.fetch_recipe_image(recipe)
+                if image is not None:
+                    file_extension = os.path.splitext(recipe.photo)[1]
```

Comment:
Thank you @coddingtonbear for the review! I'm wondering whether you prefer all this logic in a new method. As you see using the path object a lot here could be simplified